docs/dev-setup.md

```diff
@@ -110,13 +109,6 @@ To log in as one of the base data users, when prompted for a username and passwo
 1. If you want to see the app running, continue to the [React Frontend](#react-frontend) instructions.
 
 ## Testing
-### Initial Setup Test Database
-After successfully creating and building the MSSQL Docker container it is possible to create a dedicated container for testing. Use the `-PtestEnv` property to access the test environment settings in `gradle`.
-1. Create the MSSQL Docker container and test database `./gradlew -PtestEnv dockerCreateDB`
-1. Start the MSSQL Docker container: `./gradlew -PtestEnv dockerStartDB`
-1. Wait until the container is fully started (can be done automatically with `./gradlew -PtestEnv dbWait`), then run `./gradlew -PtestEnv dbMigrate`
-1. Seed initial data - MSSQL: `./gradlew -PtestEnv dbLoad`.
-1. Run `./gradlew -PtestEnv build` to download all dependencies and build the project.
```

Comment:
This setup describes setting up the test database (hence the `-PtestEnv`, which selects a different container etc.).

Reply:
Yeah, indeed I missed the point between the distinct between test-DB and the development-DB.